kafkajs/consumer.js

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2024 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+'use strict'
+const pino = require('pino')
+const logger = pino({ level: 'debug' })
+const { Kafka } = require('kafkajs')
+const kafka = new Kafka({
+  clientId: 'test-consumer',
+  brokers: ['localhost:9093']
+})
+const topics = ['test-topic', 'other-topic']
+const consumer = kafka.consumer({ groupId: 'test-group' })
+
+async function main() {
+  await consumer.connect()
+  topics.forEach(async (topic) => {
+    await consumer.subscribe({ topic })
+  })
+  await consumer.run({
+    eachMessage: async ({ topic, partition, message }) => {
+      logger.info(`Handling message from topic: ${topic}`)
```

Comment:
For consistency with line 39:

```suggestion
      logger.info('Handling message from topic: %s', topic)
```